packages/react/src/components/List/List.tsx

```diff
@@ -1,24 +1,38 @@
-import type { ComponentPropsWithoutRef } from 'react';
+import type { HTMLAttributes, ReactNode } from 'react';
 import React from 'react';
 import cn from 'classnames';
 
+import { Paragraph } from '../Typography';
+
 import classes from './List.module.css';
 
 export type ListProps = {
-  /** Select which border style between items*/
-  borderStyle?: 'solid' | 'dashed';
-} & ComponentPropsWithoutRef<'ul'>;
+  /**
+   * The type of list to render.
+   * @default ul
+   */
+  as?: 'ul' | 'ol';
+  /** Changes text sizing
+   * @default medium
+   */
+  size?: 'xsmall' | 'small' | 'medium' | 'large';
```

Comment:
I don't think we want `xsmall` for `List` do we?

Design only has small, medium and large